server/scripts/k6service.js

```diff
@@ -0,0 +1,46 @@
+import http from 'k6/http';
+import { sleep, check } from 'k6';
+import { Rate } from 'k6/metrics';
+const { getRandom, review } = require('./k6TestData.js');
+
+export let options = {
+  stages: [
+    { duration: '5s', target: 1 },
+    { duration: '25s', target: 10 },
+    { duration: '30s', target: 100 },
+    { duration: '1m', target: 300 },
+    { duration: '1m', target: 600 },
+    { duration: '1m', target: 1000 },
+    { duration: '1m', target: 1000 },
+    { duration: '1m', target: 600 },
+    { duration: '1m', target: 300 },
+    { duration: '1m', target: 0 },
+  ],
+};
+
+const base_url = 'http://localhost:3004/api/allreviews';
+let postCount = 5;
+
+export default function() {
+  let randomId = getRandom(990000, 998000);
+  let path = `${base_url}/${randomId}`;
+
+  // Test /GET
+  check(http.(path), {
+    "/GET received status 200": r => r.status === 200
+  });
+  sleep(1);
+
+  // Test /POST on every 5th turn
+  postCount--;
+  if (postCount === 0) {
+    randomId = getRandom(998000, 999000);
```

Comment:
Could you help me understand why you use getRandom(998000, 999000) here as opposed to getRandom(990000, 998000) on Line 25?

Reply:
I had wanted to use a different set of ids for my testing between the two HTTP methods. However, I'm thinking of teasing out POST tests from GET tests so that I can get more information for the specific requests.